src/main.c

```diff
@@ -45,7 +54,7 @@ int main(void) {
         x++;
     }
 
-    printf("%d\n", x);
+    LOG_INFO("Total number of interations: %d", x);
     SDL_Quit();
```

Comment:
I think we are currently leaking the calloc done by `mem_init` 😆

Reply:
That's true, good point. But that is not part of this PR and should be taken care of in another one, right?

Reply:
Ah I see what you mean. It's not `mem_init`but `cli_config_default`. That's leaking, too.

Reply:
Please have a look, but that one liner in main.c should fix that